server/graphql/requisition/src/mutations/request_requisition/insert.rs

```diff
@@ -335,10 +335,9 @@ mod test {
                     expected_delivery_date: Some(NaiveDate::from_ymd(2022, 01, 03))
                 }
             );
-            Ok(Requisition {
-                requisition_row: mock_request_draft_requisition(),
-                name_row: mock_name_a(),
-            })
+            Ok(inline_init(|r: &mut Requisition| {
+                r.requisition_row = mock_request_draft_requisition()
```

Comment:
Quite a lot of changes in this pr are from this, adding a store to requisition domain object and changing requisition initialisation to inline method